README.md

```diff
@@ -1,132 +1,135 @@
-[![CI](https://github.com/pygfx/pygfx/workflows/CI/badge.svg)](https://github.com/pygfx/pygfx/actions)
-[![Documentation Status](https://readthedocs.org/projects/pygfx/badge/?version=latest)](https://pygfx.readthedocs.io/en/latest/?badge=latest)
-[![PyPI version](https://badge.fury.io/py/pygfx.svg)](https://badge.fury.io/py/pygfx)
-
 # pygfx
+[![CI ](https://github.com/pygfx/pygfx/workflows/CI/badge.svg)
+](https://github.com/pygfx/pygfx/actions)
+[![Documentation Status
+](https://readthedocs.org/projects/pygfx/badge/?version=latest)
+](https://pygfx.readthedocs.io/en/latest/?badge=latest)
+[![PyPI version ](https://badge.fury.io/py/pygfx.svg)
+](https://badge.fury.io/py/pygfx)
+
+A python render engine targeting Vulkan/Metal/DX12.
+
+
+<p align="center">
+<img src="./docs/_static/readme_sponza.png" alt="drawing" width="200"/>
+<img src="./docs/_static/readme_pbr_example.webp" alt="drawing" width="200"/>
+<img src="./docs/_static/readme_torus_knot_wire.png" alt="drawing" width="200"/>
+</p>
+<p align="center">
+(Check out the 
+<a href="https://pygfx.readthedocs.io/en/latest/_gallery/index.html"> example 
+gallery </a> for more renders.)
+</p>
 
-A render engine, inspired by ThreeJS, but for Python and targeting Vulkan/Metal/DX12 (via wgpu).
-
-
-## Introduction
-
-This is a Python render engine build on top of [WGPU](https://github.com/pygfx/wgpu-py) (instead of OpenGL).
+## Installation
 
-We take a lot of inspiration from ThreeJS, e.g.:
+```bash
+pip install -U pygfx glfw
+```
```

Comment:
all the same caveats apply as for wgpu-py, e.g. you need pip >=20.3 on linux, etc. I think that's going to lead to the same kind of issues on this repo if we don't mention it like we do over there

Reply:
I added a link to the platform requirements for wgpu-py and a general note.